django_jinja/management/commands/makemessages.py

```diff
@@ -54,7 +54,7 @@ def handle(self, *args, **options):
         trans_real.endblock_re = re.compile(
             trans_real.endblock_re.pattern + '|' + r"""^-?\s*endtrans\s*-?$""")
         trans_real.block_re = re.compile(
-            trans_real.block_re.pattern + '|' + r"""^-?\s*trans(?:\s+(?!'|")(?=.*?)|\s*-?$)""")
+            trans_real.block_re.pattern + '|' + r"""^-?\s*trans(?:\s+(?:no)?trimmed)?(?:\s+(?!'|")(?=.*?=.*?)|\s*-?$)""")
```

Comment:
`trimmed` and `notrimmed` seem to be the only additional flags for the trans tag. https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n